content/en/tutorials/route-53-failover/index.md

```diff
@@ -214,70 +204,26 @@ test.hello-localstack.com. 300	IN	CNAME	12345.execute-api.localhost.localstack.c
+To mimic a regional outage in the `us-west-1` region, we'll configure the [Chaos Plugin]({{< ref "chaos-plugin" >}}) to halt all service invocations in this region, including the health check function.
+Once the primary region becomes non-functional, Route 53's health checks will fail.
+This failure will activate the failover policy, redirecting traffic to the corresponding services in the secondary region, thus maintaining service continuity.
 
 {{< command >}}
-$ cat region-outage-experiment.json
-<disable-copy>
-{
-    "description": "template for internal server error for few regions i.e. us-west-1",
-    "actions": {
-        "regionUnavailable-us-west-1": {
-            "actionId": "localstack:generic:api-error",
-            "parameters": {
-                "region": "us-west-1",
-                "errorCode": "503"
-            }
-        }
-    },
-    "stopConditions": [],
-    "roleArn": "arn:aws:iam:000000000000:role/ExperimentRole"
-}
-</disable-copy>               
-{{< /command >}}
-
-This Fault Injection Simulator (FIS) experiment template is set up to mimic a `Service Unavailable` (503 error) in the `us-west-1` region.
-To create the experiment template, use the following command:
-
-{{< command >}}
-$ awslocal fis create-experiment-template --cli-input-json file://region-outage-experiment.json
-{{< /command >}}
-
-Once the template is created, start the experiment using its ID:
-
-{{< command >}}
-$ awslocal fis start-experiment --experiment-template-id <EXPERIMENT_TEMPLATE_ID>
-<disable-copy>
-{
-    "experiment": {
-    "id": "651b5196-b244-4a8b-8ab6-d7b9e13998a0",
-    "experimentTemplateId": "d3a1a31b-c52e-49ec-8387-8f5eb75a11df",
-    "roleArn": "arn:aws:iam:000000000000:role/ExperimentRole",
-    "state": {
-        "status": "running"
-    },
-    "actions": {
-        "regionUnavailable-us-east-1": {
-        "actionId": "localstack:generic:api-error",
-        "parameters": {
-            "region": "us-west-1",
-            "errorCode": "503"
-            }
-        }
-    },
-    "stopConditions": [],
-    "creationTime": 1699902569.439826,
-    "startTime": 1699902569.439826
+$ curl -L -X POST 'http://localhost.localstack.cloud:4566/_localstack/chaos/faults' \
+-H 'Content-Type: application/json' \
+-d '
+[
+    {
+        "region": "us-west-1"
     }
-}
-</disable-copy>
+]'
 {{< /command >}}
 
-Replace `<EXPERIMENT_TEMPLATE_ID>` with the ID of the experiment template created in the previous step. When the experiment is active,
-Route 53's health checks will detect the failure and redirect traffic to the standby region as per the failover setup. Confirm this redirection with:
+This will cause all services to fail in the `us-west-1` region with a 503 Service Unavailable error.
+Because of this, Route 53's health checks will detect the failure and redirect traffic to the standby region as per the failover setup.
+
+Confirm this redirection with:
 
 {{< command >}}
 $ dig @localhost test.hello-localstack.com CNAME
```

Comment:
maybe a little hint on what we are seeing in this dig output, since it is mostly identical to the original output.

Reply:
Added in 50df0b445